cli/Cargo.toml

```diff
@@ -3,6 +3,19 @@ name = "lumina-cli"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
+description = "Celestia data availability node implementation in Rust"
+repository = "https://github.com/eigerco/lumina"
+# readme = "README.md" -- It's using the project's root readme as for now, should it have a separate one?
```

Comment:
how should we handle this?

Reply:
I guess root `README` could link to a separate `README` in `cli`, but I don't think it's worth it right now. So I'd vote for leaving it as is.

Reply:
there is also a symlink way